### PR TITLE
issue 116 - implemented request to get routes for ipv6 destinations

### PR DESCRIPTION
**Pull request by manuel-domke**
implemented the xml request to get static routes for ipv6 destinations from #116

### CODE REVIEW

**Comment by manuel-domke** [comment]
It seems to be impossible to improve code coverage (within scope of this project).

The test case for napalm_base only tests IPv4 at this time:
See https://github.com/napalm-automation/napalm-base/blob/develop/napalm_base/test/getters.py line 354ff:
    @wrap_test_cases
    def test_get_route_to(self, test_case):
        """Test get_route_to."""
        destination = '1.0.4.0/24'
  
Test added in napalm-automation/napalm-base#233

File: napalm_iosxr/iosxr.py [hunk 1]
```diff
@@ -1240,15 +1240,30 @@ def get_route_to(self, destination='', protocol=''):
...
-        {network}</Address>{prefix}</Naming></Route></RouteTable></IP_RIBRoute></IP_RIBRouteTable>\
-        </SAF></SAFTable></AF></AFTable></VRF></VRFTable></RIB></Operational></Get>'.format(
-            network=network,
-            prefix=prefix_tag
-        )
+        ipv = 4
+        try:
+            ipv = IPAddress(network).version
+        except AddrFormatError:
+            return {'error': 'Wrong destination IP Address!'}
```
Comment (mirceaulinic) [inline napalm_iosxr/iosxr.py]: Please raise TypeError instead or returning a dictionary.

**Review by mirceaulinic — CHANGES_REQUESTED** [review]
A small change please and we're ready to get this in.
Thanks @manuel-domke

**Review by mirceaulinic — APPROVED** [review]

**Comment by mirceaulinic** [comment]
Thanks @manuel-domke @thorko @juergen-geinitz

**Comment by mirceaulinic** [comment]
This will be included in the next version, 0.5.0. Hope to be able to release it this week.

**Pull request merged**